components/python/tornado/Makefile

```diff
@@ -54,4 +49,6 @@ REQUIRED_PACKAGES += library/python/python-mysql-27
 # Auto-generated dependencies
```

Comment:
The build dependencies look strange. Are you sure that only Python-2.7 versions are needed?